src/dto.js

```diff
@@ -141,13 +140,13 @@ var methods = [
  * @param {*}
  * @returns {*}
  */
-methods.forEach(function(method) {
-  BaseListDTO.prototype[method] = function() {
-    return [][method].apply(this.items || [], arguments);
-  };
-});
+methods.forEach(function (method) {
+  BaseListDTO.prototype[method] = function () {
+    return [][method].apply(this.items || [], arguments)
```

Comment:
could we modernize this maybe? `apply` may not be required any more using spread and stuff

Reply:
we can, but I'd prefer a separate PR for that